### PR TITLE
Make format specifier for time_t portable

### PR DESCRIPTION
**Pull request by maskit**
PRId64 doesn't work for time_t on macOS.

This page uses "%jd", and it works for me.
https://en.cppreference.com/w/c/chrono/time_t

### CODE REVIEW

**Comment by maskit** [comment]
[approve ci autest]

**Comment by traeak** [comment]
[approve ci autest]

**Comment by shinrich** [comment]
[approve ci autest]

**Comment by maskit** [comment]
Yeah, you're probably right. I didn't read the code that much and just fixed the compile errors because it blocks my daily work. I'd like to leave the future improvement to people who maintain the plugins.

File: plugins/cache_range_requests/cache_range_requests.cc [hunk 1]
```diff
@@ -479,7 +479,8 @@ handle_cache_lookup_complete(TSHttpTxn txnp, txndata *const txn_state)
   if (TS_SUCCESS == TSHttpTxnCacheLookupStatusGet(txnp, &cachestat)) {
     if (TS_CACHE_LOOKUP_HIT_FRESH == cachestat) {
       time_t const ch_time = get_date_from_cached_hdr(txnp);
-      DEBUG_LOG("IMS Cached header time %" PRId64 " vs IMS %" PRId64, ch_time, txn_state->ims_time);
+      DEBUG_LOG("IMS Cached header time %jd vs IMS %jd", static_cast<intmax_t>(ch_time),
+                static_cast<intmax_t>(txn_state->ims_time));
```
Comment (ywkaras) [inline plugins/cache_range_requests/cache_range_requests.cc:483]: If you prefer to cast to intmax_t then why not use PRIdMAX as the format specifier?  https://en.cppreference.com/w/cpp/header/cinttypes

Comment (maskit) [reply plugins/cache_range_requests/cache_range_requests.cc:483]: It's not my preference. Just because Google navigated me to the example and it worked.

**Comment by zwoop** [comment]
Backported to 9.0.1 in #7435

**Pull request merged**